great_expectations/data_context/store/tuple_store_backend.py

```diff
@@ -254,6 +254,9 @@ def __init__(  # noqa: PLR0913
         base_public_path=None,
         store_name=None,
     ) -> None:
+        if not filepath_suffix and not filepath_template:
+            filepath_suffix = ".json"
```

Comment:
The `TupleFilesystemStoreBackend` should default to `.json` (currently we have some weird logic in store constructors to set this value). I'd modify the default in the signature but some things break and this felt like the path of least resistance